pkg/kapp/diff/change.go

```diff
@@ -148,3 +159,8 @@ func (d *ChangeImpl) calculateOpsDiff() OpsDiff {
 
 	return OpsDiff(patch.Diff{Left: existingObj, Right: newObj}.Calculate())
 }
+
+func isAdoptedResource(res ctlres.Resource) bool {
```

Comment:
why is this called isAdoptedResource?

Reply:
I was thinking of using `isExternalResource`, but this scenario doesn't apply to all the external resources. Maybe we can use something like `isPlaceholderResource`. Thoughts?

Reply:
Or we can follow the existing terminology and call it `hasPlaceholderAnnotation`?

Reply:
hasPlaceholderAnnotation LGTM